src/server.py

```diff
@@ -0,0 +1,162 @@
+                    "Last-Modified: Wed, 08 Jan 2003 23:11:55 GMT\r\n" +
+                    "Etag: '3f80f-1b6-3e1cb03b'\r\n" +
+                    "Accept-Ranges:  none\r\n" +
+                    "Content-Length: 438\r\n" +
+                    "Connection: close\r\n" +
+                    "Content-Type: text/html; charset=UTF-8\r\n" +
+                    "\r\n" +
+                    "<438 bytes of content>"),
+        "method": ("HTTP/1.1 405 Method Not Allowed\r\n" +
+                    "Date: Mon, 23 May 2005 22:38:34 GMT\r\n" +
+                    "Server: Apache/1.3.3.7 (Unix) (Red-Hat/Linux)\r\n" +
+                    "Last-Modified: Wed, 08 Jan 2003 23:11:55 GMT\r\n" +
+                    "Etag: '3f80f-1b6-3e1cb03b'\r\n" +
+                    "Accept-Ranges:  none\r\n" +
+                    "Content-Length: 438\r\n" +
+                    "Connection: close\r\n" +
+                    "Content-Type: text/html; charset=UTF-8\r\n" +
+                    "\r\n" +
+                    "<438 bytes of content>"),
+        "version": ("HTTP/1.1 403 Forbidden\r\n" +
+                    "Date: Mon, 23 May 2005 22:38:34 GMT\r\n" +
+                    "Server: Apache/1.3.3.7 (Unix) (Red-Hat/Linux)\r\n" +
+                    "Last-Modified: Wed, 08 Jan 2003 23:11:55 GMT\r\n" +
+                    "Etag: '3f80f-1b6-3e1cb03b'\r\n" +
+                    "Accept-Ranges:  none\r\n" +
+                    "Content-Length: 438\r\n" +
+                    "Connection: close\r\n" +
+                    "Content-Type: text/html; charset=UTF-8\r\n" +
+                    "\r\n" +
+                    "<438 bytes of content>"),
+        "host": ("HTTP/1.1 417 Expectation Failed\r\n" +
+                    "Date: Mon, 23 May 2005 22:38:34 GMT\r\n" +
+                    "Server: Apache/1.3.3.7 (Unix) (Red-Hat/Linux)\r\n" +
+                    "Last-Modified: Wed, 08 Jan 2003 23:11:55 GMT\r\n" +
+                    "Etag: '3f80f-1b6-3e1cb03b'\r\n" +
+                    "Accept-Ranges:  none\r\n" +
+                    "Content-Length: 438\r\n" +
+                    "Connection: close\r\n" +
+                    "Content-Type: text/html; charset=UTF-8\r\n" +
+                    "\r\n" +
+                    "<438 bytes of content>"),
+        "format": ("HTTP/1.1 400 Bad Request\r\n" +
+                    "Date: Mon, 23 May 2005 22:38:34 GMT\r\n" +
+                    "Server: Apache/1.3.3.7 (Unix) (Red-Hat/Linux)\r\n" +
+                    "Last-Modified: Wed, 08 Jan 2003 23:11:55 GMT\r\n" +
+                    "Etag: '3f80f-1b6-3e1cb03b'\r\n" +
+                    "Accept-Ranges:  none\r\n" +
+                    "Content-Length: 438\r\n" +
+                    "Connection: close\r\n" +
+                    "Content-Type: text/html; charset=UTF-8\r\n" +
+                    "\r\n" +
+                    "<438 bytes of content>"),
+    }
+    for each in response_dict:
+        if key == each:
+            return response_dict[key]
```

Comment:
Why do this loop at all if you're using a dictionary to determine which error to send back? It's a dictionary. Key-value pairs. Just `return response_dict[key] if key in response_dict else response_dict['format']`